src/autopas/tuning/AutoTuner.h

```diff
@@ -382,5 +382,21 @@ class AutoTuner {
    * CSV logger for all samples collected during a tuning phase.
    */
   TuningDataLogger _tuningDataLogger;
+
+  /**
+   * Is set to true during a tuning phase.
+   */
+  bool _isTuning{false};
+
+  /**
+   * Is only set to true in the iteration where a tuning phase ends. This is done in tuneConfiguration() and set back to
+   * false in bumpIterationCounters()
+   */
+  bool _endOfTuningPhase{false};
+
+  /**
+   * Stores the number of iterations the previous tuning phase used. Is reset at the beginning of a new tuning phase.
+   */
+  size_t _iterationsInLastTuningPhase{0};
```

Comment:
Personally, I would like to rename this variable to be a bit more concise.
```suggestion
  size_t _lastTuningIterations{0};
```
Some others are all out for verbose variable names, so I think both is fine ;)

Reply:
Correct me if I'm wrong, but to me, it looks like this counter tracks the number of iterations from the most recent start of a tuning phase until the most recent end. Thus, this is not necessarily the last tuning phase but, depending on the situation the current one.

I'd maybe rename it to something like:
- `_iterationsInTuningPhase`
- `_iterationsInCurrentTuningPhase`
- `_iterationsInMostRecentTuningPhase`

(I'd prefer it to start with `_iterations` so it is more consistent with the other iteration counters)
Yes, those are longer names, but they capture better the meaning.

Also: We need to update the doc of the variable.

Reply:
I decided to go with `_iterationsInMostRecentTuningPhase`. Done in [24bbc12](https://github.com/AutoPas/AutoPas/pull/943/commits/24bbc12ae21f2cf9b58ccb144927c54eaa09c53e)